src/requestor/routes/maintain.py

```diff
@@ -97,6 +100,29 @@ async def create_request(
         logger.debug(f"Got username from token: {token_username}")
         data["username"] = token_username
 
+    if "revoke" in api_request.query_params:
+        if api_request.query_params["revoke"]:
+            raise HTTPException(
+                HTTP_400_BAD_REQUEST,
+                f"The 'revoke' parameter should not be assigned a value. Received '{api_request.query_params['revoke']}'",
+            )
+        if data.get("resource_path"):
```

Comment:
As per [line:86](https://github.com/uc-cdis/requestor/blob/cf7132c612ff013bc8b1185dae8c168f887d56fa/src/requestor/routes/maintain.py#L86), this condition never becomes True. Right?

Reply:
ah shoot, i might have to move things around